rpcs3/Emu/Cell/lv2/sys_net/lv2_socket_p2ps.cpp

```diff
@@ -858,6 +858,16 @@ void lv2_socket_p2ps::close()
 				}
 				it++;
 			}
+
+			if (p2p_port.bound_p2ps_vports.contains(vport))
+			{
+				::at32(p2p_port.bound_p2ps_vports, vport).erase(lv2_id);
+
+				if (p2p_port.bound_p2ps_vports.empty())
+				{
+					p2p_port.bound_p2ps_vports.erase(vport);
+				}
```

Comment:
If `empty()` then `erase()` ? Sounds weird...

Reply:
Good catch. It needs to che k if the ports at vport are empty